src/tasks.js

```diff
@@ -20,14 +16,12 @@ const enterTasks = getMyElement('#enter-tasks');
 const enterIcon = getMyElement('#enter-icon');
 
 function getUserData() {
+  // e.preventDefault();
```

Comment:
- [ ] Please remove commented codes as it makes your code messy and unreadable. :pray:

Reply:
You didn't make a change on this one. I will make it optional because it is minor, but please always make sure that you addressed all the reviewer comments before requesting another review. :pray: